scripts/local/resize_images.py

```diff
@@ -1,11 +1,46 @@
-# --replace flag (false by default, overwrites if same ext, deletes if diff ext)
-# Print number of files found
-# Confirmation before overwrite
-# Input directory
-# Output directory
-
-# Skip corrupt images and print their path in output
-# --trigger-size [0]
-# --max-width
-# --max-height
-# --outputExtension ()
+import os
+from utils.bulk_ops_common import (
+    get_local_argparser,
+    run_argparser,
+    walk_and_extract_files,
+    resize_image,
+)
+
+
+def resize_images(input_dir, output_dir, max_width, max_height, trigger_size):
+    """
+    Resize images in the input directory and save them in the output directory.
+
+    :param input_dir: Directory containing images to be resized.
+    :param output_dir: Directory to save resized images.
+    :param max_width: Maximum width for resizing.
+    :param max_height: Maximum height for resizing.
+    :param trigger_size: Minimum file size in bytes to trigger resizing.
+    """
+    # Extract image files from the input directory
+    file_extensions = ["jpg", "jpeg", "png", "tiff"]
```

Comment:
let's move this as an cli arg, and add this as the default value. Also add a validation that only a subset of these 4 values is supported